src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -103,7 +110,9 @@
     }
   }
 
-  function MdFabSpeedDialFlingAnimation() {
+  function MdFabSpeedDialFlingAnimation($timeout) {
+    function delayDone(done) { $timeout(done, cssAnimationDuration, false); }
+
     function runAnimation(element) {
```

Comment:
I tried switching to `$animateCSS()` during my last bout fixing bugs, but I seem to remember running into some issues and after about an hour of playing with it I reverted back to the current method.

If you think it's worth the time, I am sure I can update it, but this PR was more concerned with getting it functional again after the gestures update :-) Perhaps we should create a separate issue to track updating this?